lib/dialects/abstract/connection-manager.js

```diff
@@ -335,6 +345,15 @@ class ConnectionManager {
 
     return this.dialect.connectionManager.validate(connection);
   }
+
+  versionSupport() {
+    const version = _.get(this.sequelize, 'options.databaseVersion', 0);
+    const minVer = this.dialect.minimumSupportedVersion;
```

Comment:
`this.dialect.supports.minimumSupportedVersion`?

Reply:
My bad